srcpkgs/pinta/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'pinta'
+pkgname=pinta
+version=1.7
+revision=1
+build_style=gnu-configure
+hostmakedepends="autoconf automake pkg-config"
+makedepends="mono-devel gtk-sharp2 intltool mono-addins"
```

Comment:
`gtk-sharp2` isn't an option, we don't want new gtk2 software. This should use `gtk-sharp`.

Reply:
Okay - changed `gtk-sharp2` to `gtk-sharp`